### ISSUE TITLE
Option to use in/out markers for defining range for actions

### ISSUE THREAD

**Issue by ladlon** [turtletooth/tahoma2d#434]
In addition to my suggestions for additional selection features (Select all cells from here on, in this column... Select all cells from here on, from ALL columns...... etc), I had another potentially useful idea:

Use the in/out markers to quickly define a range for use in editing actions... like Remove all keys within markers, of the selected column...  Remove all keys within markers, of all columns...  Remove all cells within markers, etc, etc.

Basically give the in/out markers an additional use, for defining a range for editing, as well as the current playback range utilization.
